test/test_validation.jl

```diff
@@ -0,0 +1,105 @@
+import YAML
+import PowerSystems
+
+const VALID_CONFIG_FILE = joinpath(dirname(pathof(PowerSystems)),
+                                            "descriptors", "validation_config.yml")
+const WRONG_FORMAT_CONFIG_FILE = joinpath(dirname(pathof(PowerSystems)),
+                                            "descriptors", "config.yml")
+include(joinpath(DATA_DIR,"data_5bus_pu.jl"))
+
+@testset "Test reading in YAML data" begin
+    data = PowerSystems.read_validation_descriptor(VALID_CONFIG_FILE)
+    @test data isa Vector
+    @test !isempty(data)
+    function find_struct()
+        for item in data
+            if item["struct_name"] == "TechThermal"
+                return true
+            end
+        end
+    end
+    @test find_struct() == true
```

Comment:
nit:  To improve readability, in the failing case you could make find_struct return false instead of nothing and then just write ```@test find_struct"""